src/domain/Progress/ProgressTracker/ProgressTracker.examples.md

```diff
@@ -1,6 +1,10 @@
 The **Progress Tracker** is a visual representation of a users progress through a set of steps.
 They guide the user through a number of steps in order to complete a specified process.
 
+<br />
+
+Progress Tracker onclick action is only available for past step
```

Comment:
`onclick` -> \`onClick\`    (with the backticks, so it shows up in the code style)

`step` -> `steps`